application/src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/request/RaptorRequestTransferCache.java

```diff
@@ -38,7 +39,8 @@ public void put(List<List<Transfer>> transfersByStopIndex, RouteRequest request)
     final CacheKey cacheKey = new CacheKey(transfersByStopIndex, request);
     final RaptorTransferIndex raptorTransferIndex = RaptorTransferIndex.create(
```

Comment:
Make 2 methods instead of passing in a "flag" to alter the behavior:

Not:
```
public create(flagg, ...)
```
Instead:
```
public createInitialSetup(..., ) { create(SETUP, ...); }
public createRequestScope(..., ) { create(REQUEST_SCOPE, ...); }
private create(flagg, ...)
```
The enum then can be private to the implementation.

Reply:
The term in OTP to tell that something is part of a API request is `REQUEST_SCOPE`, use that instead of `RUNTIME`